src/DAMA/DoctrineTestBundle/Doctrine/DBAL/PostConnectEventListener.php

```diff
@@ -9,7 +9,9 @@ class PostConnectEventListener
     public function postConnect(ConnectionEventArgs $args): void
     {
         $connection = $args->getConnection();
-        if (!$connection->getDriver() instanceof StaticDriver) {
+        $params = $connection->getParams();
```

Comment:
unfortunately `Connection::getParams()` was marked `@internal` on DBAL. Will discuss it.